src/tasks/DecisionTask.ts

```diff
@@ -54,11 +54,12 @@ export class DecisionTask extends Task<SWF.DecisionTask> {
   setExecutionContext(context: any) {
     this.executionContext = context
   }
-  private buildTaskInput(input: any, overrideEnv?: any): string {
+  private buildTaskInput(input: any, overrideEnv?: any, control?: any): string {
```

Comment:
I am thinking it might be nice to put an interface around control.

In `src/interfaces.ts` I would just add a quick interface like:

```
export interface ControlData {
  executionCount: Number,
  maxRetry: Number
}
```

Then still keep it optional with the ?